index.bs

```diff
@@ -3076,6 +3085,41 @@ and |session|:
 
 </div>
 
+<div algorithm="locate nodes using role">
+To <dfn>locate nodes using role</dfn> with given |context nodes|, |selector|,
```

Comment:
Puppeteer would currently need both accessible name and role as an input to the query. Would it make sense to include both into this locator?

Reply:
I believe the plan is to specify a distinct locator for accessible name.

Reply:
Those are different though: they are for getting the values and not for searching. 

I think it'd would be beneficial to have a single locator to search by a11y attributes (so it's easier to extend the list of attributes). Currently, the locators not composable so the client like Puppeteer would be required to query twice and merge results client-side.

Reply:
Related https://github.com/WICG/aom/issues/197